### ISSUE TITLE
Validation Loss

### ISSUE THREAD

**Issue by gentaiscool** [iamyuanchung/Autoregressive-Predictive-Coding#3]
Hi, do you have validation loss scores for all models (n=1,...n=20) on libri-valid-clean? 
I want to verify whether my results are correct. I created a test script and loaded your pre-trained models.

I got these validation losses:
n=1, 0.30188
n=3, 0.50266

Thank you in advance.

**Comment by iamyuanchung** [comment]
The numbers you got are within a reasonable range on dev-clean. Here are my numbers for each model at epoch #100:

n = 1: 0.29454
n = 2: 0.41830
n = 3: 0.51855
n = 5: 0.66831
n = 10: 0.90817
n = 20: 1.08558

**Comment by gentaiscool** [comment]
Thank you
